### PR TITLE
fix text color in dropdown of light theme

### PR DESCRIPTION
**Pull request by connyduck**
before
device-2019-05-26-100309

after
device-2019-05-26-100139

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -31,9 +31,7 @@
     </style>
 
     <style name="AppTheme.Account.AppBarLayout" parent="ThemeOverlay.AppCompat">
-        <item name="android:textColorPrimary">@color/text_color_primary_dark</item>
-        <item name="android:textColorSecondary">@color/text_color_secondary_dark</item>
-        <item name="android:textColorTertiary">@color/text_color_tertiary_dark</item>
+        <item name="android:colorControlNormal">@color/white</item>
```
Comment (charlag) [inline app/src/main/res/values/styles.xml]: White is okay for both themes? I'm confused tbh

Comment (connyduck) [reply app/src/main/res/values/styles.xml]: Yes this only affects the ViewMediaActivity hat is completely black regardless of theme.

Comment (charlag) [reply app/src/main/res/values/styles.xml]: Okay, maybe we should document that so that we don't have surprises later

Comment (connyduck) [reply app/src/main/res/values/styles.xml]: true, lets rename the theme to make clear where its used

**Pull request merged**